upi/openstack/inventory.yaml

```diff
@@ -32,3 +27,15 @@ all:
       # Number of provisioned Compute nodes.
       # 3 is the minimum number for a fully-functional cluster.
       os_compute_nodes_number: 3
+
+      # The public network, if available.
+      os_external_network: 'external'
+
+      # OpenShift API floating IP address. If this value is non-empty, the
+      # corresponding floating IP will be attached. Additionally if non-empty,
```

Comment:
This comment needs to be updated to remove the mention of bootstrap node FIP

Reply:
Good catch. While I was there, I have added some verbosity to the documentation.